app/src/test/kotlin/co/netguru/baby/monitor/client/feature/voiceAnalysis/VoiceAnalysisUseCaseTest.kt

```diff
@@ -46,40 +46,36 @@ class VoiceAnalysisUseCaseTest {
     fun `should send noiseDetection message to baby device`() {
         voiceAnalysisUseCase.chooseVoiceAnalysisOption(
             messageController,
-            VoiceAnalysisOption.NoiseDetection
+            VoiceAnalysisOption.NOISE_DETECTION
         ).test()
             .assertComplete()
 
         verify(messageController)
-            .sendMessage(argThat { voiceAnalysisOption == VoiceAnalysisOption.NoiseDetection.name })
+            .sendMessage(argThat { voiceAnalysisOption == VoiceAnalysisOption.NOISE_DETECTION.name })
     }
 
     @Test
     fun `should send machineLearning message to baby device`() {
         voiceAnalysisUseCase.chooseVoiceAnalysisOption(
             messageController,
-            VoiceAnalysisOption.MachineLearning
+            VoiceAnalysisOption.MACHINE_LEARNING
         ).test()
             .assertComplete()
 
-        verify(messageController).sendMessage(argThat {
-            voiceAnalysisOption ==
-                    VoiceAnalysisOption.MachineLearning.name
-        })
+        verify(messageController).sendMessage(argThat { voiceAnalysisOption ==
+                VoiceAnalysisOption.MACHINE_LEARNING.name })
     }
 
     @Test
     fun `should get successful response with the same confirmationId`() {
         voiceAnalysisUseCase.chooseVoiceAnalysisOption(
             messageController,
-            VoiceAnalysisOption.MachineLearning
+            VoiceAnalysisOption.MACHINE_LEARNING
         ).test()
             .assertValue(true)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/2cb48707f0295dcdf2a5669ca721de8461628495/app/src/test/kotlin/co/netguru/baby/monitor/client/feature/voiceAnalysis/VoiceAnalysisUseCaseTest.kt#L75"></span>Unexpected indentation (6) (it should be 8)</td>
    </tr>
  </tbody>
</table>